### DISCUSSION TITLE
Working on issues

### DISCUSSION THREAD

**Discussion by divyanshugit** [optuna/optuna#3823] [Q&A] [answered]
Hi @nzw0301 and  @himkt, I'm finally settled down. So, I want to work on the things I've mentioned in the GSoC proposal. Can we have a quick call where we can discuss the proposal and I can start working on it?

**Comment by nzw0301** [comment, marked as answer]
I’m afraid I cannot do that since I’m quite busy nowadays. Please feel free to work on [issues with welcome-contribute tag](https://github.com/optuna/optuna/issues?q=is%3Aissue+is%3Aopen+label%3Acontribution-welcome).
